src/pages/FocusPage.tsx

```diff
@@ -15,14 +15,64 @@ import { FocusCompareEqualsPage } from './FocusCompareEqualsPage';
 import { FocusCompareToBaselinePage } from './FocusCompareToBaselinePage';
 import { getLocation } from '../helpers/get-location';
 import { formatVariantDisplayName } from '../data/VariantSelector';
+import { VariantSelector } from '../data/VariantSelector';
+
+type SearchType = 'aa-mutation' | 'nuc-mutation' | 'pango-lineage';
+
+type SearchOption = {
+  label: string;
+  value: string;
+  type: SearchType;
+};
 
 type Props = {
   isSmallScreen: boolean;
 };
 
+type SelectorWithId = {
+  selector: VariantSelector;
+  id: number;
+};
+
 export const FocusPage = ({ isSmallScreen }: Props) => {
   const exploreUrl = useExploreUrl()!;
 
+  const [variantQuery1, setVariantQuery1] = useState('');
+  const [selectedOptions1, setSelectedOptions1] = useState<SearchOption[]>([]);
+
+  useEffect(() => {
+    if (exploreUrl) {
+      if (exploreUrl.variant) {
+        let values: SearchOption[] = [];
+        if (exploreUrl.variant.pangoLineage) {
+          values.push({
+            label: exploreUrl.variant.pangoLineage,
+            value: exploreUrl.variant.pangoLineage,
+            type: 'pango-lineage',
+          });
+          if (exploreUrl.variant.nucMutations) {
+            for (let i of exploreUrl.variant.nucMutations) {
+              values.push({ label: i, value: i, type: 'nuc-mutation' });
+            }
+          }
+          if (exploreUrl.variant.aaMutations) {
+            for (let i of exploreUrl.variant.aaMutations) {
+              values.push({ label: i, value: i, type: 'aa-mutation' });
+            }
+          }
+        }
+        //setAdvancedSearch(false);
+        setSelectedOptions1(values);
+        setVariantQuery1('');
+      }
+      if (exploreUrl.variant?.variantQuery) {
+        //setAdvancedSearch(true);
+        setVariantQuery1(exploreUrl.variant.variantQuery);
+        setSelectedOptions1([]);
+      }
```

Comment:
This certainly does not belong into `FocusPage`. Isn't all the information already in `exploreUrl.variants` that is being passed into `VariantSearch` through the `currentSelection` parameter?